languages/php/generic/php_to_generic.ml

```diff
@@ -56,9 +56,9 @@ let ident v = wrap string v
 let var v = wrap string v
 let qualified_ident v = list ident v
 
-let name_of_qualified_ident xs =
+let name_of_qualified_ident ?(case_insensitive = false) xs =
```

Comment:
since we're in a PHP file I would set the default value to true here instead (I know you explicitely
call this function with ~case_insensitive:true below, but again because I read a conversion for PHP, I'm surprised
to see false here because I know PHP is case insensitive, so better set the default to true.

Reply:
PHP is only case insensitive for functions and classes. Of note variables are strangely not case sensitive, but fair I will switch it and make a note above the function.